File: Source/Code/Jacobi.Vst.Framework.TestPlugin/TestPluginCommandStub.cs
namespace Jacobi.Vst.Framework.TestPlugin
{
    using Jacobi.Vst.Framework;

    public class TestPluginCommandStub : Plugin.StdPluginCommandStub, Core.Plugin.IVstPluginCommandStub
    {
        protected override IVstPlugin CreatePluginInstance()
        {
            return new FxTestPlugin();
        }
    }
}

File: Source/Samples/Jacobi.Vst.Samples.Delay.VB/TestPluginCommandStub.vb
Imports Jacobi.Vst.Framework
Imports Jacobi.Vst.Framework.Plugin

Public Class TestPluginCommandStub
    Inherits StdPluginDeprecatedCommandStub
    Implements Core.Plugin.IVstPluginCommandStub

    Protected Overrides Function CreatePluginInstance() As Framework.IVstPlugin

        Return New FxTestPlugin()

    End Function

End Class
